File: test/benchmark/fib/fib.rb

def fib(n)
  if n < 2 then
    n
  else
    fib(n - 1) + fib(n - 2)
  end
end

start = Time.now
for i in 0...10
  puts fib(28)
end
puts "elapsed: " + (Time.now - start).to_s + ' s'

File: test/benchmark/fib/fib.py
from time import process_time as clock

def fib(n):
  if n < 2: return n
  return fib(n - 1) + fib(n - 2)

start = clock()
for i in range(0, 10):
  print(fib(32))
print("elapsed: ", clock() - start, 's')
